debian/clearwater-infrastructure.links

```diff
@@ -4,3 +4,7 @@
 /usr/share/clearwater/bin/sync_alarms.py /usr/bin/cw-sync_alarms
 /usr/share/clearwater/bin/clear_alarms.py /usr/bin/cw-clear_alarms
 /usr/share/clearwater/bin/restart_node_processes /usr/bin/cw-restart_node_processes
+/usr/share/clearwater/bin/clearwater-check-config /usr/sbin/clearwater-check-config
```

Comment:
Why is this in /usr/sbin?

Reply:
It requires root - things that require root should be in `/usr/sbin`.